contracts/src/v0.8/dev/ocr2dr/OCR2DRRegistry.sol

```diff
@@ -331,37 +334,28 @@ contract OCR2DRRegistry is
       revert InsufficientBalance();
     }
 
-    uint64 nonce = currentNonce + 1;
-    bytes32 requestId = computeRequestId(msg.sender, billing.client, billing.subscriptionId, nonce);
+    s_nonce++;
+    // Simplify requestId calculation
+    bytes32 requestId = keccak256(abi.encode(address(this), s_nonce));
```

Comment:
Sure, in that case we can include the address of the requester too.

Reply:
Actually, after thinking about this, maybe we should use the subscriptionId and _not_ the requester address for the hash.  This is because of the way `getRequestsToTimeOut` works.

Reply:
What is the motivation for simplifying the requestID calculation?